### PR TITLE
Add test with lowest PHPUnit version to Travis matrix

### PR DESCRIPTION
**Pull request by enov**

### CODE REVIEW

**Comment by enov** [comment]
It seems there's a version conflict here. Any ideas how to solve this?

File: .travis.yml [hunk 1]
```diff
@@ -8,8 +8,14 @@ php:
...
   - hhvm
 
+matrix:
+  include:
+    - php: 5.3
+      env: 'COMPOSER_PHPUNIT="lowest"'
+
 before_script:
   - COMPOSER_ROOT_VERSION=3.3.x-dev composer install --prefer-dist
+  - if [ "$COMPOSER_PHPUNIT" = "lowest" ]; then composer update --prefer-lowest --with-dependencies phpunit/phpunit; fi;
```
Comment (cs278) [inline .travis.yml]: Missing `COMPOSER_ROOT_VERSION=3.3.x-dev`

**Comment by acoulton** [comment]
Thanks @enov I've merged all the others, ping me when you've got this one green. I think @cs278 is right, it should just need the COMPOSER_ROOT_VERSION

**Comment by enov** [comment]
Yeah, that seems to do the trick. Thanks @cs278!
cc @acoulton

**Comment by acoulton** [comment]
Great, nice one @enov

**Pull request merged**